### PR TITLE
refactor: cleanup

### PR DESCRIPTION
**Pull request by lucasmirloup**
- remove legacy Apache 2.2
- build using Docker bake
- containers now stop immediately on SIGTERM
- stop using `jwilder/dockerize`
- lint using hadolint
- remove unused Brotli settings
- create CONTRIBUTING.md
- update README

### CODE REVIEW

**Comment by lucasmirloup** [comment]
Hi @aegypius :wave:
Feel free to take a look and comment on the changes :+1:

**Comment by lucasmirloup** [comment]
~~Note : I changed the default value of `DOCUMENT_ROOT` from `/var/www/symfony/web` to `/var/www/symfony/public` as this is [the default value since Symfony 4.x](https://symfony.com/doc/current/configuration/override_dir_structure.html#override-the-public-directory).~~

~~Developers using the updated image on Symfony <= 3.x projects will have to specify this in their `docker-compose.yml` :~~
```yaml
services:
  web:
    environment:
      DOCUMENT_ROOT: /var/www/symfony/web
```

**Comment by lucasmirloup** [comment]
I reverted the change from `/var/www/symfony/web` to `/var/www/symfony/public` in order to avoid BC breaks in this PR.

File: virtualhost.conf
Comment (thislg) [inline virtualhost.conf:1]: Put back brotli

**Review by thislg — CHANGES_REQUESTED** [review]

**Review by thislg — APPROVED** [review]

**Pull request merged**